qemu/tests/block_stream_installation.py

```diff
@@ -0,0 +1,36 @@
+import time
+import random
+
+from virttest import utils_test
+from virttest import utils_misc
+
+from qemu.tests import blk_stream
+
+
+def run(test, params, env):
+    """
+    block_stream_installation test:
+    1). guest installation
+    2). live snapshot during guest installation
+    3). block stream afterwards
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    args = (test, params, env)
+    bg = utils_misc.InterruptedThread(utils_test.run_virt_sub_test, args,
+                                      {"sub_type": "unattended_install"})
+    bg.start()
+    utils_misc.wait_for(bg.is_alive, timeout=10)
+    time.sleep(random.uniform(60, 200))
+    tag = params["source_image"]
+    stream_test = blk_stream.BlockStream(test, params, env, tag)
```

Comment:
same as above.
These are no big deals.
It just my personally think making one more assignment does not make it earlier to read.

Reply:
@hereischen Same, allow me keep it this way. :)